src/librustc_resolve/lib.rs

```diff
@@ -2194,7 +2177,9 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
     // user and one 'x' came from the macro.
     fn binding_mode_map(&mut self, pat: &Pat) -> BindingMap {
         let mut result = HashMap::new();
-        pat_bindings(&self.def_map, pat, |binding_mode, _id, sp, path1| {
+        pat_bindings(&self.def_map,
+                     pat,
+                     |binding_mode, _id, sp, path1| {
```

Comment:
the rule here is each argument starts its own line and the closure body starts a new line. This comes from a rule which basically says that if any argument spans multiple lines, then each arg must be on its own line.